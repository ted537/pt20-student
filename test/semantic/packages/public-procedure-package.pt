using input, output;

pkg publicProcedure is
    public fun procedure is 
	val x = 5;
    end;
end;
