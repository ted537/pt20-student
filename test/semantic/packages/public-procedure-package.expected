   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 1
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 2
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 4
   .tStoreInteger
     .tSkipProc
     oEmitNullAddress
     % value emitted -32767
       .tParmEnd
     .tProcedureEnd
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
